### ISSUE TITLE
Creating Keys

### ISSUE THREAD

**Issue by davinciko** [syncthing/syncthing-android#347]
Upon re-installing the app, it crashes while generating a device ID key.

**Comment by Nutomic** [comment]
Can you post a logcat?

**Comment by davinciko** [comment]
I uploaded it to my Dropbox.
https://www.dropbox.com/s/2cfrecxbhuyws9d/2015-04-09-21-36-38.txt?dl=0

**Comment by Nutomic** [comment]
Relevant part from the log:
```
04-09 21:37:08.154 I/SyncthingService(14733): Web GUI will be available at http://127.0.0.1:54027
04-09 21:37:08.234 I/WindowState(  568): WIN DEATH: Window{43a3caa0 u0 com.nutomic.syncthingandroid/com.nutomic.syncthingandroid.activities.MainActivity}
04-09 21:37:08.234 I/ActivityManager(  568): Process com.nutomic.syncthingandroid (pid 14733) has died.
04-09 21:37:08.244 W/ActivityManager(  568): Scheduling restart of crashed service com.nutomic.syncthingandroid/.syncthing.SyncthingService in 1000ms
04-09 21:37:08.244 W/ActivityManager(  568): Force removing ActivityRecord{427d32f8 u0 com.nutomic.syncthingandroid/.activities.MainActivity t232}: app died, no saved state
04-09 21:37:08.244 I/WindowState(  568): WIN DEATH: Window{43908cb8 u0 com.nutomic.syncthingandroid/com.nutomic.syncthingandroid.activities.MainActivity}
04-09 21:37:08.294 W/InputMethodWrapper(  908): Incoming session is null
04-09 21:37:08.294 W/InputMethodManagerService(  568): Got RemoteException sending setActive(false) notification to pid 14733 uid 10225
04-09 21:37:08.304 D/Zygote  (  182): Process 14733 terminated by signal (15)
```

Stackoverflow says this happens if too much work is done in the GUI thread, or if too much RAM is used. But it's weird that this never happened before.

@davinciko Is this reproducible? Any specific steps? Does it also happen if you restart the device before the re-install?

**Comment by Zillode** [comment]
Duplicate of https://github.com/syncthing/syncthing-android/issues/216
